site/content/docs/5.0/components/toasts.md

```diff
@@ -43,7 +43,7 @@ Toasts are as flexible as you need and have very little required markup. At a mi
 
 ### Live
 
-Click the button the below to show as toast (positioning with our utilities in the lower right corner) that has been hidden by default with `.hide`.
+Click the button the below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.
```

Comment:
```suggestion
Click the button below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.
```

Reply:
doh, embarassing to spot one but not the other...